v03_pipeline/lib/reference_data/compare_globals.py

```diff
@@ -33,12 +33,11 @@ def __getitem__(self, name: str):
     @classmethod
     def from_dataset_configs(
         cls,
-        rdc: ReferenceDatasetCollection,
-        dataset_type: DatasetType,
         reference_genome: ReferenceGenome,
+        datasets: list[str],
```

Comment:
I made this change in crdq pr and added to this one so that I don't make merge conflicts for myself

Reply:
Should we also change `from_ht` below to use a `datasets` list rather than the `rdc`?  Just to share an "interface"?

Reply:
yes, I like that

Reply:
that makes sense, I still want to pass `datasets` to `get_datasets_to_update` to be explicit because the code doesn't enforce that `ht1_globals` and `ht2_globals` have the same datasets

Reply:
Isn't that a good thing though?

- In production world we want a discrepancy to go noticed.
- In tests, we can force the behavior by mocking the config correctly rather than mocking the `rdc.datasets`, which I think actually simplifies things.